chaste_codegen/chaste_model.py

```diff
@@ -276,7 +279,12 @@ def _get_time_variable(self):
         self._set_is_metadata(time_variable, 'derived-quantity')
         try:
             # If the variable is in units that can be converted to millisecond, perform conversion
-            return self._model.convert_variable(time_variable, desired_units, DataDirectionFlow.INPUT)
+            time_var = self._model.convert_variable(time_variable, desired_units, DataDirectionFlow.INPUT)
+            self._model.rdf.add((time_var.rdf_identity, create_rdf_node((self._PYCMLMETA, 'derived-quantity')),
+                                create_rdf_node('yes')))
+            self._model.rdf.add((time_var.rdf_identity, create_rdf_node((self._PYCMLMETA, 'derived-quantity')),
```

Comment:
Duplicate line??

Reply:
uups yes I was testing it would be ok to do if the tag was already there.